src/components/CodeView/index.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Refractor from 'react-refractor/all';
+
+import styles from './styles.module.scss';
+
+import CodeView, { getLanguage, getLines } from '.';
+
+describe(__filename, () => {
+  const render = ({
+    content = 'some content',
+    mimeType = 'mime/type',
+  } = {}) => {
+    return shallow(<CodeView mimeType={mimeType} content={content} />);
+  };
+
+  it('renders a CodeView component', () => {
+    const root = render();
+
+    expect(root.find(`.${styles.CodeView}`)).toHaveLength(1);
+    expect(root.find(`.${styles.lineNumbers}`)).toHaveLength(1);
```

Comment:
What about adding another test that makes sure multi-line content produces multiple span tags? Like: 
```
root.find(`.${styles.lineNumbers} span`)
``` 

This would test the integration of `CodeView` and `getLines()`.